examples/js/loaders/GLTFLoader.js

```diff
@@ -1408,20 +1589,18 @@ THREE.GLTFLoader = ( function () {
 			if ( node.jointName ) {
 
 				_node = new THREE.Bone();
+				_node.name = node.name !== undefined ? node.name : node.jointName;
```

Comment:
For skeletal animation, generally bone should have non-empty name.
So I set jointName to bone.name if name in json is undefined.